doc/classes/Line2D.xml

```diff
@@ -66,37 +66,37 @@
 		</member>
-		<member name="default_color" type="Color" setter="set_default_color" getter="get_default_color">
+		<member name="default_color" type="Color" setter="set_default_color" getter="get_default_color" default="Color( 0.4, 0.5, 1, 1 )">
 			The line's color. Will not be used if a gradient is set.
 		</member>
-		<member name="end_cap_mode" type="int" setter="set_end_cap_mode" getter="get_end_cap_mode" enum="Line2D.LineCapMode">
+		<member name="end_cap_mode" type="int" setter="set_end_cap_mode" getter="get_end_cap_mode" enum="Line2D.LineCapMode" default="0">
 			Controls the style of the line's last point. Use [code]LINE_CAP_*[/code] constants. Default value: [constant LINE_CAP_NONE].
 		</member>
-		<member name="gradient" type="Gradient" setter="set_gradient" getter="get_gradient">
+		<member name="gradient" type="Gradient" setter="set_gradient" getter="get_gradient" default="null">
 			The gradient is drawn through the whole line from start to finish. The default color will not be used if a gradient is set.
 		</member>
-		<member name="joint_mode" type="int" setter="set_joint_mode" getter="get_joint_mode" enum="Line2D.LineJointMode">
+		<member name="joint_mode" type="int" setter="set_joint_mode" getter="get_joint_mode" enum="Line2D.LineJointMode" default="0">
 			The style for the points between the start and the end.
 		</member>
-		<member name="points" type="PoolVector2Array" setter="set_points" getter="get_points">
+		<member name="points" type="PoolVector2Array" setter="set_points" getter="get_points" default="PoolVector2Array(  )">
 			The points that form the lines. The line is drawn between every point set in this array.
 		</member>
-		<member name="round_precision" type="int" setter="set_round_precision" getter="get_round_precision">
+		<member name="round_precision" type="int" setter="set_round_precision" getter="get_round_precision" default="8">
 			The smoothness of the rounded joints and caps. This is only used if a cap or joint is set as round.
 		</member>
-		<member name="sharp_limit" type="float" setter="set_sharp_limit" getter="get_sharp_limit">
+		<member name="sharp_limit" type="float" setter="set_sharp_limit" getter="get_sharp_limit" default="2.0">
 			The direction difference in radians between vector points. This value is only used if [code]joint mode[/code] is set to [constant LINE_JOINT_SHARP].
 		</member>
-		<member name="texture" type="Texture" setter="set_texture" getter="get_texture">
+		<member name="texture" type="Texture" setter="set_texture" getter="get_texture" default="null">
 			The texture used for the line's texture. Uses [code]texture_mode[/code] for drawing style.
 		</member>
-		<member name="texture_mode" type="int" setter="set_texture_mode" getter="get_texture_mode" enum="Line2D.LineTextureMode">
+		<member name="texture_mode" type="int" setter="set_texture_mode" getter="get_texture_mode" enum="Line2D.LineTextureMode" default="1101004800">
```

Comment:
Looks bogus, uninitalized variable?

Reply:
Indeed, can you fix it? ([line_2d.cpp:41](https://github.com/godotengine/godot/blob/master/scene/2d/line_2d.cpp#L41), `_texture_mode = LINE_TEXTURE_NONE`)